outlines/text/generate/regex.py

```diff
@@ -210,16 +206,16 @@ def choice(model, choices: List[str], max_tokens: Optional[int] = None):
 
 
 def json(model, schema: Union[str, BaseModel], max_tokens: Optional[int] = None):
-    """Generate a text sequence that follows a JSON schema.
+    """Generate a text sequence that follows a JSON schema or Pydantic model.
 
     Parameters
     ---------
     model
-        The model to use to computes the next-token logits.
+        The model to use to compute the next-token logits.
```

Comment:
This is a nitpick, but I think these comments for `model` args might be more welcoming to some readers with less context, without losing clarity, if we said "the language model" / "the text transformer model" or something, instead of just "the model" - being that "model" is a very vague word and requires previous context to understand. For example, just in this file alone, there is a connotation of "model" as a text transformer model and "model" as a Pydantic data validation model.

Reply:
Hey @jrysana, that sounds like a good idea to me 🙂  would you want to make those changes?

Reply:
Sure, pushed @zaqqwerty